doom-modeline.el

```diff
@@ -89,59 +89,59 @@
 
 (doom-modeline-def-modeline 'main
   '(bar workspace-name window-number modals matches follow buffer-info remote-host buffer-position word-count parrot selection-info)
-  '(objed-state misc-info persp-name battery grip irc mu4e gnus github debug repl lsp minor-modes input-method indent-info buffer-encoding major-mode process vcs checker))
+  '(objed-state misc-info persp-name battery grip irc mu4e gnus github debug repl lsp minor-modes input-method indent-info buffer-encoding major-mode process vcs checker date))
```

Comment:
Not sure if it's the best position for the date segment. Can you please describe it?

Reply:
I was inspired by a[wesome-tray](https://github.com/manateelazycat/awesome-tray) which put date info to the right end of the modeline and I think it would be more nature than other place for people looking after the time 😄